src/mdacli/__main__.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+#
+# Copyright (c) 2021 Authors and contributors
+#
+# Released under the GNU Public Licence, v2 or any higher version
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+"""
+Entrypoint module, in case you use `python3 -mdacli`.
+
+
+Why does this file exist, and why __main__? For more info, read:
+
+- https://www.python.org/dev/peps/pep-0338/
+- https://docs.python.org/3/using/cmdline.html#cmdoption-m
+"""
+
+import sys
+
+if __package__ is None and not hasattr(sys, 'frozen'):
```

Comment:
Can you briefly explain me meaning of this code? Is it just to in case __main__ is called directly? That shouldn't ever happen though. `__main__` is a dunder, users are not supposed to run that file per se. If some one wants to run it, it should be his/her responsibility. We should not facilitate such use. :wink:

Reply:
Yes thats the reason but if this never happens we can remove this here :)